content/en/blog/_posts/2021-08-16-minreadysecond-statefulsets.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: 'minReadySeconds for StatefulSets'
```

Comment:
Is there a typical format for the titles of post-release blog articles? If so, let's follow that.

Reply:
I am not sure of that. Where do we have style guide?

Reply:
@jlbutler ^?

Reply:
Apologies I thought I'd replied to this. As I mentioned last week, I also think this could use a more reader-friendly title. 

I don't know of any specific guideline but they do tend to be more descriptive of functionality, rather than calling out details.